Xamarin.Forms.Platform.MacOS/ModalPageTracker.cs

```diff
@@ -41,6 +41,16 @@ public Task<Page> PopAsync(bool animated)
 			return HideModalAsync(modal, animated);
 		}
 
+		internal void LayoutSubviews()
+		{
+			foreach(var modal in _modals)
```

Comment:
Can we add a check to see if _renderder.View is not null or something like that?